Adding additional persistence locations targeted by malware.

vonnopsled opened the pull request:
These are mostly for older versions, but some less common malware persistence locations.

<!-- Reviewable:start -->
[<img src="https://reviewable.io/review_button.png" height=40 alt="Review on Reviewable"/>](https://reviewable.io/reviews/forensicartifacts/artifacts/95)
<!-- Reviewable:end -->

joachimmetz commented:
Overall looks ready to go in, a few small comments.

---

Review status: 0 of 2 files reviewed at latest revision, 4 unresolved discussions, all commit checks successful.

---

<sup>**[definitions/webbrowser.yaml, line 311 \[r1\]](https://reviewable.io:443/reviews/forensicartifacts/artifacts/95#-K-7Wr-7bCHh1qCS2jaG)** ([raw file](https://github.com/forensicartifacts/artifacts/blob/0f7f37690d4c7a526f5ea55eb67d55742e78da09/definitions/webbrowser.yaml#L311)):</sup>
IMO no need to repeat MSIE here so I opt:
Applies to MSIE versions 7-11 => Applies to versions 7 - 11

---

<sup>**[definitions/webbrowser.yaml, line 320 \[r1\]](https://reviewable.io:443/reviews/forensicartifacts/artifacts/95#-K-7W4yDNhid4zn4pOny)** ([raw file](https://github.com/forensicartifacts/artifacts/blob/0f7f37690d4c7a526f5ea55eb67d55742e78da09/definitions/webbrowser.yaml#L320)):</sup>
I opt: MSIE6 => Microsoft Internet Explorer 6

---

<sup>**[definitions/windows.yaml, line 21 \[r1\]](https://reviewable.io:443/reviews/forensicartifacts/artifacts/95#-K-7Wcy2Oudngcx9OUd3)** ([raw file](https://github.com/forensicartifacts/artifacts/blob/0f7f37690d4c7a526f5ea55eb67d55742e78da09/definitions/windows.yaml#L21)):</sup>
KEY_ => HKEY_

---

<sup>**[definitions/windows.yaml, line 87 \[r1\]](https://reviewable.io:443/reviews/forensicartifacts/artifacts/95#-K-7VxBI6HAq5oqrpXWT)** ([raw file](https://github.com/forensicartifacts/artifacts/blob/0f7f37690d4c7a526f5ea55eb67d55742e78da09/definitions/windows.yaml#L87)):</sup>
I opt: GUID => Class identifier (CLSID)

---


Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/forensicartifacts/artifacts/95)
<!-- Sent from Reviewable.io -->

vonnopsled commented:
Updated with suggested corrections. Please take a look.

---

Review status: 0 of 2 files reviewed at latest revision, 1 unresolved discussion, some commit checks pending.

---

<sup>**[definitions/windows.yaml, line 87 \[r1\]](https://reviewable.io:443/reviews/forensicartifacts/artifacts/95#-K-7VxBI6HAq5oqrpXWT)** ([raw file](https://github.com/forensicartifacts/artifacts/blob/0f7f37690d4c7a526f5ea55eb67d55742e78da09/definitions/windows.yaml#L87)):</sup>
Sounds good. Updated to Class identifier (CLSID).

---


Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/forensicartifacts/artifacts/95)
<!-- Sent from Reviewable.io -->

joachimmetz commented:
LGTM, merging.
